benchmarks/docs/setup/prometheus-operator-values.yml

```diff
@@ -9,6 +9,9 @@ grafana:
     userKey: admin-user
     passwordKey: admin-password
   grafana.ini:
+    server:
+      # REPLACE THIS WITH THE ACTUAL ROOT URL
+      root_url: "http://localhost:3000"
```

Comment:
I could use some suggestions here. You need to set the root URL correctly as otherwise the GitHub authentication will not work. If you configure nothing, then it will use `localhost:3000`, which it will pass to GitHub as the "redirect_uri", and GitHub will reject authentication calls saying it doesn't match what's configured in the OAuth app.

We could hardcode the right value, but since we use the same values file for each cluster, this has the danger than we overwrite the URL in one cluster or the other. Any ideas? I'd like to keep it simple so we can keep using the same file, but I don't know. Maybe I missed some config option?

Reply:
not sure whether it get it. So you have to set here the ingress URL?

Reply:
Yes, here it should be the ingress URL for each ingress that we have.

Reply:
So one of the issue I mentioned was this: if we leave it to localhost:3000 and someone upgrades without changing it, then we break the OAuth. But if we leave it to a particular URL, then we risk breaking the OAuth for one of the two Grafana we have.

I'm thinking of looking into Helmfile to manage our multiple deployments, sharing the same file with some overrides.